client.go

```diff
@@ -93,31 +90,10 @@ func (c *Client) Production() *Client {
 // transparently before sending the notification. It will return a Response
 // indicating whether the notification was accepted or rejected by the APNs
 // gateway, or an error if something goes wrong.
+//
+// It wraps PushWithCtx for back compatibility.
 func (c *Client) Push(n *Notification) (*Response, error) {
```

Comment:
@sideshow How important is API stability for this project? Do you want to keep the no-context version around?